extensions/example-extension/page.tsx

```diff
@@ -1,13 +1,14 @@
-import { Component, LensRendererExtension } from "@k8slens/renderer-extensions";
+import { LensRendererExtension } from "@k8slens/core-extensions";
+import { Component } from "@k8slens/renderer-extensions";
```

Comment:
Import of `LensRendererExtension` still comes from `core` package but keeps name about renderer..
Not too obvious when there are many packages from `@k8slens/*`

Maybe we could make single package import like `@k8slens/extension` for both `main` and `render` processes and provide all types from there, but depending on process not everything would be available in runtime (just like `electron` serving everything from single package, `app`/`ipcMain`/etc. only for main but from same pack).

Reply:
@ixrock I like this idea... let's see if we can make it work.